pyquil/unitary_tools.py

```diff
@@ -276,6 +276,17 @@ def lifted_gate(gate: Gate, n_qubits: int):
     else:
         matrix = QUANTUM_GATES[gate.name]
 
+    for mod in gate.modifiers:
```

Comment:
It doesn't matter for these two modifiers, but you might want to walk through `gate.modifiers` in reverse order.